site/en/docs/webstore/images/index.md

```diff
@@ -239,31 +238,20 @@ thumb for designing your images:
 
 The following graphics are examples of the promotional images for an extension:
 
-<table>
-  <thead>
-    <tr>
-      <th>Small promo image (440x280)</th>
-      <th>Large promo image (920x680)</th>
-    </tr>
-  </thead>
-  <tbody>
-    <tr>
-      <td>
-        {% Img src="image/nPi5hOhvGtatFrSN6ugwlPVHcs43/9KruJ8pLUNTrzXIXP6JN.png", 
-               alt="Small promo image", height="140", width="220" %}
-      </td>
-      <td>
-        {% Img src="image/nPi5hOhvGtatFrSN6ugwlPVHcs43/uvSRznxKJngidMKDIeqN.png", 
-               alt="Large promo image", height="340", width="460" %}
-      </td>
-    </tr>
-  </tbody>
-</table>
+<figure>
+{% Img src="image/nPi5hOhvGtatFrSN6ugwlPVHcs43/9KruJ8pLUNTrzXIXP6JN.png", alt="Small promo image", height="140", width="220" %}
+  <figcaption>
+Small promo image (440x280)
+  </figcaption>
+</figure>
```

Comment:
I've been meaning to ask you an Ian to start making sure you're using `<figure>` and `<figcaption>`.

Reply:
Good idea 💯 .